sdk/bootstrap/main.go

```diff
@@ -305,6 +314,51 @@ func installDCOSCertIntoJRE() {
 	log.Println("Successfully installed the certificate.")
 }
 
+func decodeKeystores() error {
```

Comment:
Completed: 1, 3

I am picking from go gradle documentation:

> The gradle-golang plugin is is designed to give you the power of a professional build tool (in this case Gradle) to build, test (and other great stuff) with your go code. And this without pain of think about the correct installation go sdk, set the right environment variables, download dependencies, ...

> No pre installed go toolchain is required - only a working JDK (in minimum version 1.7).
> Choice to build in the default system GOPATH or in a temporary one to do not influence your build result

It looks like the plugin could be a replacement for our custom `golang` build scripts and it would be easier setting up dev environment as no `go` installation is required.